### PR TITLE
Add Commit to Travis per @pfefferle deployment script

### PR DESCRIPTION
**Pull request by dshanske**
Getting ready for simple deployment

### CODE REVIEW

**Review by snarfed — COMMENTED** [review]
i don't know travis, so I'll defer to @pfefferle, but...

File: .travis.yml
Comment (snarfed) [inline .travis.yml]: this is sketchy. we should probably add it to the repo (and review it) instead.

File: .travis.yml
Comment (snarfed) [inline .travis.yml]: is this a token for something? if so it shouldn't be in git anywhere, right? please revoke it, generate a new one, and put that token somewhere else. 😁

Comment (dshanske) [reply .travis.yml]: https://docs.travis-ci.com/user/encryption-keys/ - Only Travis can decrypt it.

File: .travis.yml
Comment (aaronpk) [reply .travis.yml]: I added it to my repo for Aperture because I didn't like the idea of running code from someone else's repo that might change at a moments notice. This is how the terrible npm hacks happen so I would like to avoid that.

File: .travis.yml
Comment (snarfed) [reply .travis.yml]: oh wow, ok! please add a comment and docs link. thanks!

**Comment by dshanske** [comment]
Going to defer until @pfefferle comments as this was originally his implementation.

File: .travis.yml
Comment (pfefferle) [reply .travis.yml]: what user have you used for the commit?

Comment (dshanske) [reply .travis.yml]: That would be my credentials right now. But snarfed wants to move to the Indieweb repo, so we should change to them

File: .travis.yml
Comment (dshanske) [reply .travis.yml]: I think he's talking about the deployment script, but okay.

Comment (snarfed) [reply .travis.yml]: (kidding! this one would be understandable, and not as painful as eg semantic linkbacks. i still vote for bundling though. 😁)

Comment (dshanske) [reply .travis.yml]: After I merge the script, how do you feel about a release? Do we need more user feedback?

Comment (snarfed) [reply .travis.yml]: i think we still have outstanding user reports of bugs and things not working, right? we should solicit more testers first, and then make sure all of those reports are fixed.

Comment (dshanske) [reply .travis.yml]: I am waiting for information from some people

Comment (dshanske) [reply .travis.yml]: But I want to be ready to go

**Review by pfefferle — DISMISSED** [review]
I tried to sum up all the feedback:

* Change the commit user to indieweb:
  * change the user in the travis file (should I make a pull request to your branch or have you access to the indieweb 1password group?)
  * add the indieweb user as commiter for the WordPress.org directory
* Move the deploy script to the local bin folder
* Add `push.sh` to the `.svnignore` file

**Comment by dshanske** [comment]
I've added the indieweb user to the readme, which once there is a push, it can push the entire repo. I'd suggest we retire the push.sh file.  I'll refactor this. I don't have access to the indieweb 1password group.

**Comment by pfefferle** [comment]
@dshanske @snarfed don't forget to add the indieweb user as admin for the WordPress.org repo.

**Comment by miklb** [comment]
does this still need that info updated in the deploy script?

**Comment by dshanske** [comment]
I don't see any reason why we can't merge this now that we've addressed the local deployment issue. Any final thoughts?

**Comment by dshanske** [comment]
As no objections noted, merged.

**Pull request merged**
